pkg/cli/cluster_free_disk_space.go

```diff
@@ -0,0 +1,125 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	rookcli "github.com/rook/rook/pkg/client/clientset/versioned"
+
+	clusterspace "github.com/replicatedhq/kurl/pkg/cluster/space"
+)
+
+const (
+	IsDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-class"
+	OpenEBSLocalProvisioner         = "openebs.io/local"
+	RookRBDProvisioner              = "rook-ceph.rbd.csi.ceph.com"
+	RookCephFSProvisioner           = "rook-ceph.cephfs.csi.ceph.com"
+)
+
+func NewClusterFreeDiskSpaceCmd(cli CLI) *cobra.Command {
```

Comment:
Would be possible doc the `NewClusterFreeDiskSpaceCmd ` func?

Reply:
fixed.